packages/ui/src/lib/screen/EmptyScreen.svelte

```diff
@@ -37,7 +37,7 @@ let copyTextDivElement: HTMLDivElement;
 </script>
 
 <div
-  class="flex flex-row w-full h-full justify-center {$$props.class || ''}"
+  class="flex flex-row w-full h-full justify-center bg-[var(--pd-details-bg)] {$$props.class || ''}"
```

Comment:
EmptyScreen is a shared component, and this modifies the background on all of them. e.g. 'breaks' empty screen on Containers/Images/Pods pages

Reply:
Ops, fixed 👍